### PR TITLE
Update names.txt

### PR DESCRIPTION
**Pull request by maryamjafari-kh**
add my name

### CODE REVIEW

**Comment by lenis03** [comment]
Hi, I hope you are doing well.
Thank you for using our Git course, and I hope you have been satisfied with the course so far.
To merge your pull request, you just need to add your first name to the end of the file 🙏
Thank you very much.

File: names.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Hadi Hajihosseini 😎🙏🏻
 Hadi
-
+maryam jafari
```
Comment (lenis03) [inline names.txt:3]: Hello, please just add your first name to the end of the file!

**Review by lenis03 — CHANGES_REQUESTED** [review]

**Review by AliPrj — APPROVED** [review]
❤️👏🏻

**Comment by AliPrj** [comment]
💪🏻👏🏻❤️🤝🏻

**Pull request closed without merging**